### PR TITLE
vote: add Issif as approvers for k8saudit-eks plugin

### PR DESCRIPTION
**Pull request by Issif**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

/kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area plugins

> /area registry

> /area build

> /area documentation

**What this PR does / why we need it**:

I'm proposing myself as approver of the `k8saudit-eks` plugin, as I developed it, it seems logical to be able to review proposals from the community.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Issif** [comment]
@leogr @mstemm @jasondellaluce Do you vote for?

File: plugins/k8saudit-eks/README.md [hunk 1]
```diff
@@ -125,22 +125,22 @@ A string which contains the name of your EKS Cluster (required).
 
 The `k8saudit-eks` plugin ships with a no default rule for test purpose, the same rules than those for `k8saudit` plugin can work, just add an `alternative` field:
 ```
-- required_engine_version: 15
+- required_engine_version: 13
```
[the remark this answers is not in the document]
Comment (Issif) [reply plugins/k8saudit-eks/README.md]: Just noticed and fixed, let me revert.

**Comment by leogr** [comment]
Ofc, +1 for me 🤗

**Review by jasondellaluce — COMMENTED** [review]
+1

**Review by leogr — APPROVED** [review]
Approved! :partying_face: (since we have a majority and no further votes cannot change the outcome)

**Pull request merged**